DOCUMENTATION.md

```diff
@@ -1,15 +1,29 @@
 # Documentation
 
-## Flatseal permissions
-
-This section indicates the permissions of Flatpak with better integration with Flatseal. It is based on the official [Sandbox Permissions Reference](https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html).
+## Table of contents
+
+- [Flatpak permissions](#flatpak-permissions)
+	- [Permissions](#permissions)
+	- [Share](#share)
+	- [Socket](#socket)
+	- [Device](#device)
+	- [Allow](#allow)
+	- [Filesystem](#filesystem)
+	- [Persistent](#persistent)
+	- [Environment](#environment)
+	- [System Bus](#system-bus)
+	- [Session Bus](#system-bus)
+- [Tips and Tricks](#tips-and-tricks)
+	- [Manually reset Flatseal permissions](#manually-reset-flatseal-permissions)
+	- [Enable custom installations](#enable-custom-installations)
+	- [Flatpak 1.7.1 or newer](#flatpak-171-or-newer)
+	- [All versions](#all-versions)
```

Comment:
This is subsection of Enable custom~, let's remove it.

Reply:
https://github.com/tchx84/Flatseal/pull/171/commits/458355b48851ac9999f217affd74829c7a2731e8